### PR TITLE
Patch 17

### PR DESCRIPTION
**Pull request by TimotejMC**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/goldzonecz/manifest.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "direct_ip": "mc.gold-zone.cz",
+    "server_wildcards": [
+        "%.gold-zone.cz"
+    ],
+    "supported_languages": [
+        "cz",
+        "sk"
+    ],
+    "social": {
+        "web": "https://www.gold-zone.cz/",
```
Comment (JakobBruening) [inline minecraft_servers/goldzonecz/manifest.json:13]: Remove "/" at the end.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/goldzonecz/manifest.json (hunk 1 above)
Comment (TimotejMC) [reply minecraft_servers/goldzonecz/manifest.json]: Done <3

Comment (TimotejMC) [reply minecraft_servers/goldzonecz/manifest.json]: How long does approval take?

**Review by JakobBruening — APPROVED** [review]
Thank you!

**Pull request merged**